src/main/java/org/interledger/ilqp/client/IlqpQuoteService.java

```diff
@@ -15,33 +14,44 @@
 import org.interledger.ilp.client.LedgerClient;
 import org.interledger.ilp.client.exceptions.ResponseTimeoutException;
 import org.interledger.ilp.client.model.ClientLedgerMessage;
-import org.interledger.ilp.core.InterledgerAddress;
-import org.interledger.ilp.core.exceptions.InterledgerQuotingProtocolException;
-import org.interledger.ilp.core.ledger.model.LedgerMessage;
+import org.interledger.ilp.InterledgerAddress;
+import org.interledger.ilp.exceptions.InterledgerQuotingProtocolException;
+import org.interledger.ilp.ledger.model.LedgerMessage;
 import org.interledger.ilqp.client.model.ClientQuoteResponse;
-import org.interledger.ilqp.core.QuoteService;
-import org.interledger.ilqp.core.model.QuoteErrorResponse;
-import org.interledger.ilqp.core.model.QuoteRequest;
-import org.interledger.ilqp.core.model.QuoteResponse;
+import org.interledger.quoting.QuoteService;
+import org.interledger.quoting.QuoteSelectionStrategy;
+import org.interledger.quoting.model.QuoteErrorResponse;
+import org.interledger.quoting.model.QuoteRequest;
+import org.interledger.quoting.model.QuoteResponse;
 
-public class ClientQuoteService implements QuoteService {
+/**
+ * The ILQP service gets quotes by using the ILQP protocol
+ * 
+ * This involves sending a quote request to neighboring connectors via the ledger interface.
+ * 
+ * As such the service is dependent on the LedgerClient
+ * 
+ * @author adrianhopebailie
+ *
+ */
+public class IlqpQuoteService implements QuoteService {
   
   private LedgerClient client;
   private Set<InterledgerAddress> connectors;
   private int requestTimeout;
   
-  public ClientQuoteService(LedgerClient client) {
+  public IlqpQuoteService(LedgerClient client) {
     this(client, client.getRecommendedConnectors(), 60 * 1000);
   }
 
-  public ClientQuoteService(LedgerClient client, Set<InterledgerAddress> connectors, int requestTimeout) {
+  public IlqpQuoteService(LedgerClient client, Set<InterledgerAddress> connectors, int requestTimeout) {
```

Comment:
might be personal preference, but any time related params i try to suffix with the unit involved, so requestTimeoutMillis. that way there can be no confusion later when queries time out in 30 millis instead of 30000 :)